extensions/sql-migration/src/wizard/summaryPage.ts

```diff
@@ -163,6 +163,16 @@ export class SummaryPage extends MigrationWizardPage {
 					constants.SHIR,
 					this.migrationStateModel._nodeNames.join(', ')));
 		}
+
+
+		this.wizard.registerNavigationValidator(async (pageChangeInfo) => {
+			this.wizard.message = { text: '' };
+			if (pageChangeInfo.newPage < pageChangeInfo.lastPage) {
```

Comment:
this can simply return true, page check doesn't change anything

Reply:
Fixed